reader-osm/src/test/resources/com/graphhopper/reader/osm/test-avoid-loops2.xml

```diff
@@ -0,0 +1,47 @@
+<?xml version='1.0' encoding='UTF-8'?>
+<osm version="0.6" generator="pbf2osm">
+
+    <!--
+      C - D
+       \ /
+    A - B - E
+    -->
+
+    <node id="1" lat="11" lon="50" uid="A">
+        <tag k="name" v="A"/>
+    </node>
+    <node id="2" lat="12" lon="51" uid="B">
+        <tag k="name" v="B"/>
+    </node>
+    <node id="2" lat="12" lon="51" uid="B2">
```

Comment:
Is it on purpose that the id 2 is used twice here ? The way below references a node 6 that is not in the data.

Reply:
Oh, will fix.